demo/src/javascripts/components/demos/trips-layer/trips-layer.js

```diff
@@ -1,7 +1,8 @@
-import {Layer, assembleShaders} from '../../../../../../index';
 import {Model, Program, Geometry, glGetDebugInfo} from 'luma.gl';
+import {readFileSync} from 'fs';
+import {join} from 'path';
 
-const glslify = require('glslify');
+import {Layer, assembleShaders} from '../../../../../../index';
```

Comment:
I find it interesting to separate vendors modules from project source files, this way we know for sure what belongs is a dependency or not. Might be obvious right now because of the relative url but I would like to use absolute one in a near future, so I'm just laying to fondations.